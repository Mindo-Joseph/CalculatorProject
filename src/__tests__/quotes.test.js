import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';

import Quotes from '../components/Quote';

describe('Quotes', () => {
  test('renders the quotes component correctly', () => {
    const quote = renderer.create(
      <BrowserRouter>
        <Quotes />
      </BrowserRouter>,
    ).toJSON();
    expect(quote).toMatchSnapshot();
  });
});
