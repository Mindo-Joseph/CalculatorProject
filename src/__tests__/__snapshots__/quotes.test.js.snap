// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quotes renders the quotes component correctly 1`] = `
<div>
  <div
    className="header"
  >
    <h1>
      Math Magicians
    </h1>
    <div
      className="navbar"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Home 
      </a>
      <a
        href="/calculator"
        onClick={[Function]}
      >
        Calculator 
      </a>
      <a
        href="/quote"
        onClick={[Function]}
      >
        Quote 
      </a>
    </div>
  </div>
  <h3>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
  </h3>
</div>
`;
